File: server-scala-play-server/app/model/UserActivityResponse.scala
package model

import play.api.libs.json._
import java.time.OffsetDateTime

/**
  * Represents the Swagger definition for UserActivityResponse.
  */
@javax.annotation.Generated(value = Array("org.openapitools.codegen.languages.ScalaPlayFrameworkServerCodegen"), date = "2026-02-09T20:57:39.376804970Z[Etc/UTC]", comments = "Generator version: 7.20.0-SNAPSHOT")
case class UserActivityResponse(
  valid: Option[Boolean],
  message: Option[String],
  version: Option[Double],
  serializeNulls: Option[Boolean],
  startTimeLog: Option[Long],
  errorCode: Option[String],
  request: Option[List[NameStringValueResponse]],
  created: Option[OffsetDateTime],
  latitude: Option[Double],
  longitude: Option[Double],
  tag: Option[String],
  customId: Option[Long],
  customType: Option[String],
  userId: Option[Long],
  username: Option[String],
  appTitle: Option[String],
  appId: Option[Long],
  appUrl: Option[String],
  appDescription: Option[String],
  imageURL: Option[String],
  profileImage: Option[String],
  fields: Option[String],
  id: Option[Long],
  customText: Option[String],
  customValue: Option[Double],
  customMessage: Option[String],
  returning: Option[String]
)

object UserActivityResponse {
  implicit lazy val userActivityResponseJsonFormat: Format[UserActivityResponse] = Json.format[UserActivityResponse]
}

File: server-aspnet-fastendpoints/src/Org.OpenAPITools/Models/UserActivityResponse.cs
namespace Org.OpenAPITools.Models;


/// <summary>
/// 
/// </summary>
public class UserActivityResponse 
{
    public bool Valid { get; set; }
    public string Message { get; set; }
    public double VarVersion { get; set; }
    public bool SerializeNulls { get; set; }
    public long StartTimeLog { get; set; }
    public string ErrorCode { get; set; }
    public List<NameStringValueResponse> Request { get; set; }
    public DateTime Created { get; set; }
    public double Latitude { get; set; }
    public double Longitude { get; set; }
    public string Tag { get; set; }
    public long CustomId { get; set; }
    public string CustomType { get; set; }
    public long UserId { get; set; }
    public string Username { get; set; }
    public string AppTitle { get; set; }
    public long AppId { get; set; }
    public string AppUrl { get; set; }
    public string AppDescription { get; set; }
    public string ImageURL { get; set; }
    public string ProfileImage { get; set; }
    public string Fields { get; set; }
    public long Id { get; set; }
    public string CustomText { get; set; }
    public double CustomValue { get; set; }
    public string CustomMessage { get; set; }
    public string Returning { get; set; }
}


